packages/demo-app/src/styles/_examples.scss

```diff
@@ -0,0 +1,141 @@
+@import "~@blueprintjs/core/lib/css/blueprint-modern.css";
```

Comment:
I don't think you need this import; you're already importing this stylesheet in `index.scss`